styleguide/components/31-molecules/accordion/_accordion.scss

```diff
@@ -0,0 +1,77 @@
+.accordion {
+
+  .accordion--button {
+    @include icon(chevron-down);
+    position: relative;
+    padding-right: 1.6em;
+    padding-left: 0;
+    color: inherit;
+
+    &::before {
+      position: absolute;
+      right: 0;
+      transition: transform .2s ease-in-out;
+      font-size: 1.6rem;
+      line-height: .9;
+    }
+
+    &[aria-expanded=true] {
+      &::before {
+        transform: scaleY(-1);
+      }
+    }
+  }
+
+  .accordion--content {
+    transition: max-height .5s ease-in-out;
+    overflow: hidden;
+  }
+
+}
+
+ul.accordion {
+  margin: 0;
+
+  li {
+    list-style: none;
+  }
+}
+
+dl.accordion {
+  dd {
+    padding: 0;
+  }
+}
+
+ul.accordion,
+dl.accordion {
+
+  .accordion--button {
```

Comment:
Other `&--button` syntax

Reply:
doens't match with the first selectors
that'll give me ul.accordion--button